docs/ABOUT.md

```diff
@@ -0,0 +1,5 @@
+_The ECMAScript specification is a standardized specification of a scripting language developed by Brendan Eich of Netscape; initially it was named Mocha, later LiveScript, and finally JavaScript._ --[Wikipedia](https://en.wikipedia.org/wiki/ECMAScript)
+
+_ECMAScript has grown to be one of the world’s most widely used general purpose programming languages. It is best known as the language embedded in web browsers but has also been widely adopted for server and embedded applications._ --[ECMA International Language Specification](http://www.ecma-international.org/ecma-262/6.0/index.html#sec-ecmascript-overview)
+
+The current revision of ECMAScript is ECMAScript 2015, also known as ES6 or Harmony.  This version brings many enhancements over ES5, including notably template strings, expressive arrow function syntax, and cleaner syntax for defining classes.  Support in current browsers is incomplete, and often requires transpilation with a tool like [Babel](https://babeljs.io/).   
```

Comment:
Despite of *..., also known as ES6...* I would say *..., previously known as ES6...*